packages/mux-player/src/media-theme-mux/media-theme-mux.ts

```diff
@@ -0,0 +1,269 @@
+  <div slot="centered-chrome" class="mxp-center-controls">
+    ${MediaSeekBackwardButton(props)}
+    ${MediaPlayButton()}
+    ${MediaSeekForwardButton(props)}
+  </div>
+  <media-control-bar>
+    <media-time-range></media-time-range>
+    <mxp-time-display></mxp-time-display>
+    ${MediaMuteButton()}
+    <media-volume-range></media-volume-range>
+    <media-playback-rate-button></media-playback-rate-button>
+    ${MediaFullscreenButton()}
+  </media-control-bar>
+`;
+
+// prettier-ignore
+export const VodChromeLarge = (props: ThemeMuxTemplateProps) => html`
+  <div slot="centered-chrome" class="mxp-center-controls">
+    ${MediaPlayButton()}
+  </div>
+  <media-control-bar>
+    ${MediaPlayButton()}
+    ${MediaSeekBackwardButton(props)}
+    ${MediaSeekForwardButton(props)}
+    <media-time-range></media-time-range>
+    <mxp-time-display></mxp-time-display>
+    ${MediaMuteButton()}
+    <media-volume-range></media-volume-range>
+    <media-playback-rate-button></media-playback-rate-button>
+    ${props.hasCaptions ? MediaCaptionsButton(props) : html``}
+    ${MediaAirplayButton()}
+    ${MediaPipButton()}
+    ${MediaFullscreenButton()}
+  </media-control-bar>
+`;
+
+// prettier-ignore
+export const LiveChromeExtraSmall = VodChromeExtraSmall;
+
+// prettier-ignore
+export const LiveChromeSmall = (props: ThemeMuxTemplateProps) => html`
+  <media-control-bar slot="top-chrome">
+    <slot name="seek-to-live-button"></slot>
+    ${Spacer()}
+    ${props.hasCaptions ? MediaCaptionsButton(props) : html``}
+    ${MediaAirplayButton()}
+    ${MediaPipButton()}
+  </media-control-bar>
+  <div slot="centered-chrome" class="mxp-center-controls">
+    ${MediaPlayButton()}
+  </div>
+  <media-control-bar>
+    ${MediaMuteButton()}
+    <media-volume-range></media-volume-range>
+    ${Spacer()}
+    ${MediaFullscreenButton()}
+  </media-control-bar>
+`;
+
+// prettier-ignore
+export const LiveChromeLarge = (props: ThemeMuxTemplateProps) => html`
+  <media-control-bar slot="top-chrome">
+    <slot name="seek-to-live-button"></slot>
+  </media-control-bar>
+  <div slot="centered-chrome" class="mxp-center-controls">
+    ${MediaPlayButton()}
+  </div>
+  <media-control-bar>
+    ${MediaMuteButton()}
+    <media-volume-range></media-volume-range>
+    ${Spacer()}
+    ${props.hasCaptions ? MediaCaptionsButton(props) : html``}
+    ${MediaAirplayButton()}
+    ${MediaPipButton()}
+    ${MediaFullscreenButton()}
+  </media-control-bar>
+`;
+
+function getProps(el: MediaThemeMux, state?: any): ThemeMuxTemplateProps {
+  return {
+    streamType: el.getAttribute('stream-type'),
+    playerSize: el.getAttribute('player-size'),
+    defaultHiddenCaptions: el.hasAttribute('default-hidden-captions'),
+    hasCaptions: el.hasAttribute('has-captions'),
+    forwardSeekOffset: el.getAttribute('forward-seek-offset'),
+    backwardSeekOffset: el.getAttribute('backward-seek-offset'),
+    ...state,
+  };
+}
+
+class MediaThemeMux extends HTMLElement {
```

Comment:
non-blocking: Per our conversation, we will should refactor this to extend a "`MediaTheme`" that we expect to minimally have a `render()` method that will be invoked on initialization to anticipate integration with Media Chrome.

Reply:
Media Chrome PR here https://github.com/muxinc/media-chrome/pull/209